backend/src/models/Settings.js

```diff
@@ -0,0 +1,9 @@
+import mongoose from 'mongoose';
+
+const settingsSchema = new mongoose.Schema({
+  appTheme: { type: String, enum: ['light', 'dark'], default: 'light' },
```

Comment:
You can extract this to constants - it will be easier to maintain
eg. 
const LIGHT_THEME = 'light';

This approach helps to eliminate typos in different places of the app (eg. in initializer)